### PR TITLE
Add checks for definition of signing algorithm field

### PR DESCRIPTION
**Pull request by ezelkow1**
Currently the logs get spammed when pulling remap (or other) files. Need to make sure that the signed field uses a defined check for the signing algorithm string before assigning a value.  We also need to check defined before using it to test for the url_sig plugin text

### CODE REVIEW

File: traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm [hunk 1]
```diff
@@ -2574,11 +2574,14 @@ sub build_remap_line {
 	if ( defined( $remap->{edge_header_rewrite} ) ) {
 		$text .= " \@plugin=header_rewrite.so \@pparam=" . $remap->{hdr_rw_file};
 	}
-	if ( $remap->{signing_algorithm} eq "url_sig" ) {
-		$text .= " \@plugin=url_sig.so \@pparam=url_sig_" . $remap->{ds_xml_id} . ".config";
-	}
-	elsif ( $remap->{signing_algorithm} eq "uri_signing" ) {
-		$text .= " \@plugin=uri_signing.so \@pparam=uri_signing_" . $remap->{ds_xml_id} . ".config";
+	if ( defined( $remap->{signing_algorithm}))
+	{
```
Comment (PSUdaemon) [inline traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm]: You are breaking the formatting of the surrounding code by putting your curly on a new line.

File: traffic_ops/app/lib/UI/ConfigFiles.pm [hunk 2]
```diff
@@ -1048,11 +1048,14 @@ sub build_remap_line {
 	if ( defined( $remap->{edge_header_rewrite} ) ) {
 		$text .= " \@plugin=header_rewrite.so \@pparam=" . $remap->{hdr_rw_file};
 	}
-	if ( $remap->{signing_algorithm} eq "url_sig" ) {
-		$text .= " \@plugin=url_sig.so \@pparam=url_sig_" . $remap->{ds_xml_id} . ".config";
-	}
-	elsif ( $remap->{signing_algorithm} eq "uri_signing" ) {
-		$text .= " \@plugin=uri_signing.so \@pparam=uri_signing_" . $remap->{ds_xml_id} . ".config";
+	if ( defined($remap->{signing_algorithm}))
+	{
```
Comment (PSUdaemon) [inline traffic_ops/app/lib/UI/ConfigFiles.pm]: Same as above.

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by ezelkow1** [comment]
fixed

**Pull request merged**